modules/hub/src/CRAuthManager.ts

```diff
@@ -43,7 +43,7 @@ export class MemoryCRAuthManager implements CRAuthManager {
     const hash = this.sha3(`${MemoryCRAuthManager.HASH_PREAMBLE} ${this.sha3(nonce)} ${this.sha3(origin)}`)
     const sigAddr = this.extractAddress(hash, signature)
 
-    if (!sigAddr || sigAddr.toLowerCase() !== address.toLowerCase()) {
+    if (!sigAddr || sigAddr !== address) {
```

Comment:
Do these toLowerCase casts happen elsewhere then?

Reply:
Yeah, we didn't have to do this inside the Spank hub, so we need to make sure we are doing it on the wallet side.